examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/templates/afv2frag-template/.content.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+    jcr:primaryType="cq:Template">
+    <jcr:content
+        cq:templateType="/conf/core-components-examples/settings/wcm/template-types/af2-fragment"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="afv2frag template"
```

Comment:
Can you please change the title of the fragment ?

Reply:
this is examples, but alright